### PR TITLE
[3134] Fix pagination not showing

### PR DESCRIPTION
**Pull request by asmega**
### Context

- Pagination options are missing from results pages under certain conditions

### Changes proposed in this pull request

- This add pagination buttons (next/previous) to bottom of results
- The root cause of this regression is not yet known
- This is fixed by explicity calculating total pages and passing to
kaminari view helper
- Fixed deprecation warning

### Guidance to review

- visit http://localhost:3002/results?fulltime=False&hasvacancies=False&l=3&parttime=False&qualifications=QtsOnly%2CPgdePgceWithQts%2COther&query=Essex+and+Thames+Primary+SCITT&senCourses=False
- there are more than 10 courses so should seen pagination options and able to paginate to second page

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [X] Cleaned commit history
- [X] Tested by running locally
- [x] Product review

### CODE REVIEW

**Comment by asmega** [comment]
[Pagination and course count is incorrect on Prod](https://trello.com/c/GZw4a6U3/3134-pagination-and-course-count-is-incorrect)

**Comment by dankmitchell** [comment]
@defong that's happening on LIVE now anyway so I would address outside of this PR i.e. https://www2.find-postgraduate-teacher-training.service.gov.uk/results?fulltime=True&hasvacancies=False&l=3&page=2&parttime=False&qualifications=QtsOnly%2CPgdePgceWithQts%2COther&query=Essex+and+Thames+Primary+SCITT&senCourses=False

File: app/view_objects/results_view.rb [hunk 1]
```diff
@@ -147,14 +147,22 @@ def courses
...
-                     .per(10)
+                     .per(per_page)
                  end
   end
 
   def course_count
     courses.meta["count"]
   end
 
+  def per_page
```
Comment (defong) [inline app/view_objects/results_view.rb]: pointless method?
`ITEMS_PER_PAGE = 10`

Comment (dankmitchell) [reply app/view_objects/results_view.rb]: It's being used below (line 163) @defong...

Comment (defong) [reply app/view_objects/results_view.rb]: https://github.com/DFE-Digital/find-teacher-training/blob/d739a4e37e575ca1a49f7fe446fab664217ab409/app/view_objects/results_view.rb#L7-L11

Its a static value so pointless method in the sense that its a constant like the others?

@dankmitchell 

https://kirshatrov.com/2019/08/20/methods-vs-constants/

Comment (asmega) [reply app/view_objects/results_view.rb]: this is now a constant but referenced in the method. it is a also now a private method

**Pull request merged**